test/color/test_hls.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+
+import kornia
+import kornia.testing as utils  # test utils
+from test.common import device_type
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+
+class TestRgbToHls:
+
+    def test_rgb_to_hls(self):
+
+        data = torch.tensor([[[21., 22.],
+                              [22., 22.]],
+
+                             [[13., 14.],
+                              [14., 14.]],
+
+                             [[8., 8.],
+                              [8., 8.]]])
+
+        expected = torch.tensor([[[0.0641, 0.07138],
```

Comment:
From where did you get this expected values. Worth mentioning it in a comment

Reply:
Okay great I'll check the changes and push a commit.
 I got the expected values from an online color convertor, I don't know is that okay or no? if yes do I add a comment line with the link of the website?

Reply:
ideally we should be able to provide a reference from where our implementation comes from. Then we can justify the unit tests in a more robust way :D

Reply:
pardon my confusion, can you clarify more about the reference thing, where or how to mention them :D

Reply:
you can add it as comment in rgb_to_hls or in the documentation. This is a not very common transform so at least a minim of documentation stating what's the algorithm behind that